proxy/common.go

```diff
@@ -25,42 +23,6 @@ var (
 	Log = common.Log
 )
 
-func callWeave(args ...string) ([]byte, []byte, error) {
```

Comment:
Are there parts of the `shell` script we can now get the rid of thanks to this refactoring?